deploy.yaml

```diff
@@ -0,0 +1,1631 @@
+apiVersion: v1
```

Comment:
is this file required?

Reply:
thanks @snaiksat , this must have been added by mistake. fixing